include/http/request_handler.js

```diff
@@ -498,6 +498,32 @@ module.exports = function RequestHandlerModule(pb) {
                     RequestHandler.redirectHosts[oldHostname] = site.hostname;
                 });
             }
+            if (pb.config.server.x_powered_by) {
+                this.resp.setHeader('x-powered-by', pb.config.server.x_powered_by);
+            }
+            this.resp.setHeader('content-type', contentType);
+            //Start: This code is to handle the "set-cookies" 
+            const secureCookies = this.req && this.req.siteObj && this.req.siteObj.secureCookies;
```

Comment:
This line will check the global setting(true/false), which is opted by client from admin panel.